### PR TITLE
chore(starters): add gatsby-starter-sasslan

### PR DESCRIPTION
**Pull request by a2zarslaan**
**Sasslan**
A minimalistic Gatsby starter template featuring SASS and CSS 7-1 architecture. The project focusses on creating websites which are not only fast, but pretty too!

**What Features this Includes**
1. SASS in the form of SCSS
2. CSS 7-1 architecture along with easy to change variables (for colors, highlight and hover effects)
3. Blog component that uses GraphQL to read and render local markdown posts with images
4. GraphQL IDE for better development workflows
5. Desktop-first and responsive design
6. Ability to use Google fonts
7. Ability to use SVG sprites
8. Github pages integration with script, quick deploy script

### CODE REVIEW

**Comment by pvdz** [comment]
Please update the description. Even if it repeats the title, that'd be better than this. But a little more information would be nice as well. Thanks!

**Comment by a2zarslaan** [comment]
Hi Peter!

I have updated the description as requested. I used npm, can you please guide me on further steps to pass the test? Looking forward to your response!

File: docs/starters.yml [hunk 1]
```diff
@@ -6300,6 +6300,38 @@
...
+    - CSS Modules
+    - React Helmet
+    - SASS
+    - Page Transitions
+    - SVG
+    - Google Fonts
+    - Responsive Design
+    - Github Pages
+    - Landing Page
+    - Beautiful typography
```
Comment (gillkyle) [inline docs/starters.yml:6320]:
```suggestion
    - Blog
    - Portfolio
    - Markdown
```

These tags are supported in our whitelist

**Comment by a2zarslaan** [comment]
Thank you so much for the help Kyle! I’ll definitely read more about Gatsby recipes as it sounds really interesting to me. Looking forward to contributing as much as I can!

**Pull request merged**